src/index.html

```diff
@@ -11,6 +11,20 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form">
+      <input
+        data-qa="big"
+        class="search__bar search__bar--big"
```

Comment:
```suggestion
        class="search__bar"
```
set all styles for search__bar and change just necessary for search__bar--small